nursery/set-system-properties.yml

```diff
@@ -0,0 +1,17 @@
+rule:
+  meta:
+    name: set system properties
```

Comment:
this loads a file path / driver, see https://is.muni.cz/th/y8opj/Bc.pdf
rename the rule and file?

Reply:
ah, ok, i'll move that over to `install driver`